codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/BuilderGeneratorBase.java

```diff
@@ -288,7 +288,6 @@ protected void renderBuildersForOperation(OperationShape operation, Shape inputS
     /**
      * @param inputShape inputShape from a streaming operation to render for.
      */
-    // TODO: should probably be in Rest Base
```

Comment:
Should it?

Reply:
I believe it should be in the base - its used by all protocols at this point and is unrelated to REST.